src/xrGame/UIGameCustom_script.cpp

```diff
@@ -13,6 +13,8 @@ SCRIPT_EXPORT(CUIGameCustom, (CDialogHolder), {
                          .def("wnd", &StaticDrawableWrapper::wnd),
 
         class_<CUIGameCustom, CDialogHolder>("CUIGameCustom")
+            .def("TopInputReceiver", &CUIGameCustom::TopInputReceiver)
```

Comment:
This function is already exported: https://github.com/OpenXRay/xray-16/blob/dev/src/xrGame/ui/UIWindow_script.cpp#L26

It should not be exported here.

Reply:
Removed